lib/repo_tools.pm

```diff
@@ -56,8 +57,9 @@ Helper to add QA web repository repository.
 This repository is *not* mandatory.
 =cut
 sub add_qa_web_repo {
-    my $repo = get_var('QA_WEB_REPO');
-    zypper_ar($repo, name => 'qa-web', no_gpg_check => 1) if ($repo);
+    my $repo   = get_var('QA_WEB_REPO');
+    my $usable = is_sle("<12") ? 0 : 1;
+    zypper_ar($repo, name => 'qa-web', no_gpg_check => $usable) if ($repo);
```

Comment:
Use `is_sle("<12")` directly (without `$usable` might be good enough for single use, but it's up to you (as `$usable` does not describe it much anyway).

Reply:
He could actually use `zypper_ar($repo, name => 'qa-web', no_gpg_check => is_sle('<12') ? undef : 1)` to overcome the issue. Still, the fix for the `zypper_ar` is nice to have :smile: